src/cohere/client.py

```diff
@@ -4,6 +4,7 @@
 
 from .base_client import BaseCohere, AsyncBaseCohere
 from .environment import ClientEnvironment
+from .manually_maintained.cache import CacheMixin
```

Comment:
I'm hoping for a better name for `manually_maintained`. @billytrend-cohere any suggestions?

Reply:
looks fine to me, we can always revisit since it's not public